### PR TITLE
fix: numaflow-go udf example & docs

### PR DESCRIPTION
**Pull request by chromevoid**
**Fix numaflow golang udf example and the `Golang` link**
image

**Fix docs**
Before:
image
image
Now:
image

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -5,11 +5,15 @@ Install Numaflow and run a couple of example pipelines.
 ## Prerequisites
 
 A Kubernetes cluster is needed to try out Numaflow. A simple way to create a local cluster is using Docker Desktop.
-* https://www.docker.com/
-* https://www.docker.com/blog/how-kubernetes-works-under-the-hood-with-docker-desktop/
+<ul>
+  <li><a href="https://www.docker.com/">Docker</a></li>
```
Comment (vigith) [inline docs/quick-start.md:9]: won't `[text](url)` style work?, something like
* [foo](foo.example.com)
* [bar](bar.example.com)

Comment (chromevoid) [reply docs/quick-start.md:9]: unfortunately no..

image
image

Comment (whynowy) [reply docs/quick-start.md:9]: Should not use HTML tags in the markdown.

Comment (chromevoid) [reply docs/quick-start.md:9]: Should we abandon the list style?

Comment (chromevoid) [reply docs/quick-start.md]: discussed offline..
thank you @whynowy for the tips

need one more blank line to render the markdown style list

**Review by whynowy — APPROVED** [review]

**Pull request merged**